pkg/datastore/postgres/repositories/header_repository_test.go

```diff
@@ -133,80 +135,6 @@ var _ = Describe("Block header repository", func() {
 		})
 	})
 
-	Describe("creating a receipt", func() {
```

Comment:
Is this being removed because we have tests in pkg/datastore/postgres/repositories/receipt_repository_test.go?

Reply:
yep 👍